frontend/components/CellInput.js

```diff
@@ -1,7 +1,19 @@
 import { html, useState, useEffect, useRef } from "../common/Preact.js"
-
 import { utf8index_to_ut16index } from "../common/UnicodeTools.js"
 
+// For customization, see: https://codemirror.net/doc/manual.html#config
+// Note that some options will require you to add additional addons in editor.html
+const custom_codemirror_options = {
```

Comment:
Why does this make customization easier?

Reply:
Maybe add a comment explaining how people can customize these? (A fork?)

Reply:
Right now it's just a separate object containing settings the user should be able to change (e.g. don't change `mode: "julia"`), but I'm not sure yet about the implementation (forking isn't ideal b/c every update will give merge conflicts), maybe a settings menu that persists to localstorage?